tools/jarjar/jarjar.bzl

```diff
@@ -65,10 +65,10 @@ def _jarjar_library(ctx):
     ctx.actions.run_shell(
         command = command,
         inputs = [
```

Comment:
let's collapse into one line: `inputs = [ctx.outputs._rules_file] + jar_files + ctx.files._jdk`?

Reply:
@ronshapiro fixed in https://github.com/google/bazel-common/pull/45/commits/3b2df5cab2e28e59d719eb51792bec937b05d08a